api/src/main/java/com/strategyobject/substrateclient/api/pallet/council/Council.java

```diff
@@ -0,0 +1,113 @@
+package com.strategyobject.substrateclient.api.pallet.council;
+
+
+import com.strategyobject.substrateclient.common.types.Result;
+import com.strategyobject.substrateclient.common.types.Unit;
+import com.strategyobject.substrateclient.pallet.annotation.Event;
+import com.strategyobject.substrateclient.pallet.annotation.Pallet;
+import com.strategyobject.substrateclient.rpc.api.AccountId;
+import com.strategyobject.substrateclient.rpc.api.primitives.Hash;
+import com.strategyobject.substrateclient.rpc.api.runtime.DispatchError;
+import com.strategyobject.substrateclient.scale.ScaleType;
+import com.strategyobject.substrateclient.scale.annotation.Scale;
+import com.strategyobject.substrateclient.scale.annotation.ScaleReader;
+import lombok.Getter;
+import lombok.Setter;
+
+@Pallet("Council")
+public interface Council {
+    /**
+     * A motion (given hash) has been proposed (by given account) with a threshold (given MemberCount).
+     */
+    @Event(index = 0)
+    @Getter
+    @Setter
+    @ScaleReader
+    class Proposed {
+        private AccountId account;
+        @Scale(ScaleType.U32.class)
+        private Long proposalIndex;
+        private Hash proposalHash;
```

Comment:
I believe we talked about not using the Hash type provided and instead using a Byte[]? Since there is a ByteReader but Hash doesn't seem to have and Scale stuff for it to decode properly from. @pfrank13

Reply:
Yeah could use some clarity on this; I thought we were mapping anonymous byte arrays as ByteArray

Reply:
Huh, never noticed this but I think the Hash usage is okay actually. He has it autoregistered `@AutoRegister(types = {Hash256.class, Hash.class, BlockHash.class, CallHash.class})`, previously I didn't think it could be deserialized because the Hash256Reader only work on Hash256 object, apparently that is incorrect.